airbyte-integrations/connectors/source-tempo/source_tempo/tempo.yaml

```diff
@@ -0,0 +1,109 @@
+version: "0.1.0"
+
+definitions:
+  schema_loader:
+    file_path: "./source_tempo/schemas/{{ options['name'] }}.json"
```

Comment:
is it some default which can be removed from YAML ?

Reply:
correct, removed it